app/models/self_disclosure_request.rb

```diff
@@ -8,22 +8,15 @@ class SelfDisclosureRequest < ApplicationRecord
 
   validates :job_application_id, uniqueness: true
 
-  def self.create_all!(job_applications)
-    job_applications.map do
-      find_or_create_by!(job_application_id: it.id).tap(&:manual!)
-    end
+  def self.create_all!(job_application)
```

Comment:
Since this doesn't operate anymore over a collection but a single job application... 
```suggestion
  def self.create_for!(job_application)
```
(or any other naming)